thanos_bot.py

```diff
@@ -1,26 +1,27 @@
-from irc import *
-from secrets import *
-import os
-import random
-import time
+from irc import IRC
+from secrets import get_channel, get_server
+
 
 channel = get_channel()
 server = get_server()
 nickname = "THANOS"
 
-irc = IRC()
-irc.connect(server, channel, nickname)
+irc_obj = IRC()
+irc_obj.connect(server, channel, nickname)
 
 
-while 1:
-    text = irc.get_text()
+while True:
+    text = irc_obj.get_text()
     print text
 
-    if "PRIVMSG" in text and channel in text and "perfectly balanced" in text.lower():
-        irc.send(channel, "As all things should be")
+    if "PRIVMSG" in text and channel in text and "perfectly balanced"\
+       in text.lower():
+        irc_obj.send(channel, "As all things should be")
 
-    if "PRIVMSG" in text and channel in text and "fun" in text.lower():
-        irc.send(channel, "Fun isn't something one considers when balancing the universe")
+    if "PRIVMSG" in text and channel in text and "fun"\
+       in text.lower():
+        irc_obj.send(channel, "Fun isn't something one considers when balancing the universe")
 
-    if "PRIVMSG" in text and channel in text and "you should have gone for the head" in text.lower():
-        irc.send(channel, "*snap*")
+    if "PRIVMSG" in text and channel in text and\
+       "you should have gone for the head" in text.lower():
```

Comment:
Since some of these conditional statements get pretty long, it could make sense to get a function that returns the result of the "PRIVMSG" and channel check. Could be called something like `check_msg()`

Reply:
def check_msg(text):
    return "PRIVMSG" in text and channel in text

Reply:
You can indent if you use code blocks. Gotta use triple back-tick (like it is in markdown)

Reply:
def check_msg(text):
>> return "PRIVMSG" in text and channel in text